opencog/nlp/chatbot-psi/chatbot.scm

```diff
@@ -18,51 +18,62 @@
     (cancel-all-threads)
     (reset-all-states)
 
-    (let* ((sent-node (car (nlp-parse utterance)))
-           (list-of-words (get-word-list sent-node)))
-
-        (State input-utterance
-            (Reference
-                list-of-words
-                sent-node
-                (Node utterance)
-            )
+    (catch #t
```

Comment:
what exceptions are being thrown? Normally, the file name and the line number are printed as a part of the exception message. do you have some examples?